velox/expression/Expr.cpp

```diff
@@ -1196,7 +1245,13 @@ void Expr::evalAll(
   }
   bool tryPeelArgs = deterministic_ ? true : false;
   bool defaultNulls = vectorFunction_->isDefaultNullBehavior();
-
+  // A null argument can mask an error in a previous argument if
+  // 1. errors are not already ignored 2. the function has default
```

Comment:
I did not understand this, it will be hard probably for future people working on the code to understand this